packages/design-system/src/components/slider/stories/index.js

```diff
@@ -37,27 +36,34 @@ const Container = styled.div`
 export default {
   title: 'DesignSystem/Components/Slider',
   component: Slider,
+  args: {
+    thumbSize: 24,
+    min: 0,
+    max: 100,
+    majorStep: 10,
+    minorStep: 1,
+  },
+  parameters: {
+    controls: {
+      exclude: ['handleChange', 'value', 'suffix'],
+    },
+  },
 };
 
-export const _default = () => {
+export const _default = (args) => {
   const [lightValue, setLightValue] = useState(0);
   const [darkValue, setDarkValue] = useState(0);
-  const thumbSize = number('Thumb size', 24);
-  const min = number('Min value', 0);
-  const max = number('Max value', 100);
-  const majorStep = number('Major step', 10);
-  const minorStep = number('Minor step', 1);
 
   return (
     <>
       <Container>
         <Text>{'Percentage:'}</Text>
         <Slider
-          thumbSize={thumbSize}
-          min={min}
-          max={max}
-          majorStep={majorStep}
-          minorStep={minorStep}
+          thumbSize={args.thumbSize}
+          min={args.min}
+          max={args.max}
+          majorStep={args.majorStep}
+          minorStep={args.minorStep}
```

Comment:
💭 nit: couldn't this have `args` spread here since the props have the same names as defined in the controls?
```suggestion
          {...args}
```

Reply:
yea.